internal/integration/emitter.go

```diff
@@ -4,19 +4,9 @@
 package integration
```

Comment:
We have the scraper in a separate package, does it make sense to separate emitters as well?
Does it require a lot of work?

Reply:
shouldn't be too hard. I'll change it